### PR TITLE
JOSS manuscript

### PR DESCRIPTION
**Pull request by musslick**
# Description

Initial submission of the manuscript for JOSS. 

# Questions (Optional)
- Please double-check your affiliations. Are they correct?

# Remarks (Optional)
- Note that we still need to add one other reference. 
- We need to make sure that the paper compiles as described here: https://joss.readthedocs.io/en/latest/paper.html#checking-that-your-paper-compiles

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+# Acknowledgements
+The AutoRA framework is being developed and maintained by members of the Autonomous Empirical Research Group. S. M., B. A., C. C. W., J. T. S. H., and J. S. were supported by the Carney BRAINSTORM program at Brown University. S. M. also received support from Schmidt Science Fellows, in partnership with the Rhodes Trust. The development of auxiliary packages for AutoRA, such as `autodoc`, is supported by the Virtual Institute for Scientific Software. The AutoRA package was developed using computational resources and services at the Center for Computation and Visualization, Brown University.
```
Comment (younesStrittmatter) [inline paper/paper.md:86]:
```suggestion
The AutoRA framework is being developed and maintained by members of the Autonomous Empirical Research Group. S. M., B. A., C. C. W., J. T. S. H., and Y. S. were supported by the Carney BRAINSTORM program at Brown University. S. M. also received support from Schmidt Science Fellows, in partnership with the Rhodes Trust. The development of auxiliary packages for AutoRA, such as `autodoc`, is supported by the Virtual Institute for Scientific Software. The AutoRA package was developed using computational resources and services at the Center for Computation and Visualization, Brown University.
```

**Review by younesStrittmatter — CHANGES_REQUESTED** [review]

File: paper/paper.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  - name: Marina Dubova
+    orcid: orcid: 0000-0001-5264-0489
+    affiliation: 5
+  - name: George T. Dang
+    affiliation: 1
+  - name: Younes Strittmatter
+    orcid: 0000-0002-3414-2838
+    equal-contrib: true
+    affiliation: 1
+  - name: John G. Holland
```
Comment (hollandjg) [inline paper/paper.md:35]:
```suggestion
  - name: John G. Holland
    orcid: 0000-0001-6845-8657
```

File: paper/paper.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+# Acknowledgements
+The AutoRA framework is being developed and maintained by members of the Autonomous Empirical Research Group. S. M., B. A., C. C. W., J. T. S. H., and Y. S. were supported by the Carney BRAINSTORM program at Brown University. S. M. also received support from Schmidt Science Fellows, in partnership with the Rhodes Trust. The development of auxiliary packages for AutoRA, such as `autodoc`, is supported by the Virtual Institute for Scientific Software. The AutoRA package was developed using computational resources and services at the Center for Computation and Visualization, Brown University.
```
Comment (hollandjg) [inline paper/paper.md:86]:
```suggestion
The AutoRA framework is developed and maintained by members of the Autonomous Empirical Research Group. S. M., B. A., C. C. W., J. T. S. H., and Y. S. were supported by the Carney BRAINSTORM program at Brown University. S. M. also received support from Schmidt Science Fellows, in partnership with the Rhodes Trust. The development of auxiliary packages for AutoRA, such as `autodoc`, is supported by the Virtual Institute for Scientific Software. The AutoRA package was developed using computational resources and services at the Center for Computation and Visualization, Brown University.
```

File: paper/paper.bib [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+@inproceedings{musslick_evaluation_2023,
+	title = {An evaluation of experimental sampling strategies for autonomous empirical research in cognitive science},
+	volume = {45},
+	author = {Musslick, Sebastian and Hewson, Joshua TS and Andrew, Benjamin W and Strittmatter, Younes and Williams, Chad C and Dang, George T and Dubova, Marina and Holland, John Gerrard},
+	year = {2023},
+	note = {Issue: 45},
+}
+
```
Comment (younesStrittmatter) [inline paper/paper.bib:15]: The capitalization of the articles seems inconsistent. "Bayesian {Machine} {Scientist}..." uses the capitalization of the original, while "An evaluation of experimental sampling strategies..." is all lowercase.

**Review by younesStrittmatter — CHANGES_REQUESTED** [review]

File: paper/paper.bib (hunk 4 above)
Comment (musslick) [reply paper/paper.bib:15]: Should now be addressed.

File: paper/paper.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  - closed-loop behavioral science
+authors:
+  - name: Sebastian Musslick
+    orcid: 0000-0002-8896-639X
+    affiliation: "1, 2"
+    corresponding: true
+  - name: Benjamin Andrew
+    affiliation: 1
+  - name: Chad C. Williams
+    orcid: orcid: 0000-0003-2016-5614
```
Comment (younesStrittmatter) [inline paper/paper.md:17]:
```suggestion
    orcid: 0000-0003-2016-5614
```

File: paper/paper.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+    affiliation: 1
+  - name: Joshua T. S. Hewson
+    affiliation: 1
+  - name: Star Li
+    orcid: orcid: 0009-0009-7849-1923
```
Comment (younesStrittmatter) [inline paper/paper.md:22]:
```suggestion
    orcid: 0009-0009-7849-1923
```

File: paper/paper.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+    affiliation: 3
+  - name: Ioana Marinescu
+    affiliation: 4
+  - name: Marina Dubova
+    orcid: orcid: 0000-0001-5264-0489
```
Comment (younesStrittmatter) [inline paper/paper.md:27]:
```suggestion
    orcid: 0000-0001-5264-0489
```

**Review by younesStrittmatter — CHANGES_REQUESTED** [review]

**Review by hollandjg — APPROVED** [review]

**Review by younesStrittmatter — APPROVED** [review]
Great!

**Pull request merged**